src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -17,10 +17,10 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/deprecation-contracts": "^2.1",
+        "symfony/deprecation-contracts": "^2.1|^3",
         "symfony/error-handler": "^4.4|^5.0|^6.0",
         "symfony/event-dispatcher": "^5.0|^6.0",
-        "symfony/http-client-contracts": "^1.1|^2",
+        "symfony/http-client-contracts": "^1.1|^2|^3",
```

Comment:
why is the http client contracts a required dependency of HttpKernel ? This looks weird to me.

Reply:
Indeed, we could remove it.

Reply:
Because of `HttpClientKernel` I suppose, but should be removed indeed

Reply:
Moved to `require-dev` because the interfaces are used to create mocks.